### PR TITLE
chore: Update test environments 6.8.15

### PR DESCRIPTION
**Pull request by kuisathaverat**
Update test environment versions to the correct Elastic Stack version.

Merge after the release 6.8.14.

Merge after first [6.8.15-SNAPSHOT Docker image](https://artifacts-api.elastic.co/v1/versions/6.8.15-SNAPSHOT)

### CODE REVIEW

File: testing/environments/latest.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.9.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.8.14
```
Comment (urso) [inline testing/environments/latest.yml:6]: These should be 7.x versions.

Comment (kuisathaverat) [reply testing/environments/latest.yml:6]:
```suggestion
    image: docker.elastic.co/elasticsearch/elasticsearch:7.11.0
```

File: testing/environments/latest.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ services:
       - "xpack.security.enabled=false"
 
   logstash:
-    image: docker.elastic.co/logstash/logstash:7.9.2
+    image: docker.elastic.co/logstash/logstash:6.8.14
```
Comment (kuisathaverat) [inline testing/environments/latest.yml:19]:
```suggestion
    image: docker.elastic.co/logstash/logstash:7.11.0
```

File: testing/environments/latest.yml [hunk 3]
```diff
@@ -26,7 +26,7 @@ services:
       - ./docker/logstash/pki:/etc/pki:ro
 
   kibana:
-    image: docker.elastic.co/kibana/kibana:7.9.2
+    image: docker.elastic.co/kibana/kibana:6.8.14
```
Comment (kuisathaverat) [inline testing/environments/latest.yml:29]:
```suggestion
    image: docker.elastic.co/kibana/kibana:7.11.0
```

**Review by v1v — APPROVED** [review]

**Review by urso — APPROVED** [review]

**Comment by kuisathaverat** [comment]
superseded by https://github.com/elastic/beats/pull/24685

**Pull request closed without merging**